config.properties.template

```diff
@@ -77,6 +78,9 @@ sort_by_iv=false
 # Always throw curveballs
 always_curve=false
 
+# Never use berries
+never_use_berries=true
```

Comment:
There is no hardcoded ball order. It calculates it by the catch probabilities it receives directly from the server.